zetta_utils/__init__.py

```diff
@@ -61,3 +61,6 @@ def load_training_modules():
     )
     from .layer import volumetric
     from .layer.volumetric import cloudvol
+
+
+load_all_modules()
```

Comment:
This should be `try_load_train_inference`

Reply:
Thanks! Fixed.